Source/WebKit/UIProcess/WebFrameProxy.cpp

```diff
@@ -536,6 +536,11 @@ void WebFrameProxy::setProcess(FrameProcess& process)
     m_frameProcess = process;
 }
 
+void WebFrameProxy::removeChildFrames()
+{
+    m_childFrames.clear();
```

Comment:
Maybe this is fine for now, but shouldn't we keep the same-origin child frames?

Reply:
Once back/forward cache is implemented we'll need to do something with them, but not keep them attached to the drawing area.